app/models/concerns/relators.rb

```diff
@@ -3,6 +3,7 @@ module Metadata
     {
       actor:                                 RDF::Vocab::MARCRelators.act,
       adapter:                               RDF::Vocab::MARCRelators.adp,
+      advisor:                               RDF::Vocab::MARCRelators.dgs,
```

Comment:
Can we call it degree_supervisor instead? That's what the actual predicate is labeled?